.bumpversion_client.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.100.5-dev0
+current_version = v0.100.5.dev1
```

Comment:
should the `v` be here?

Reply:
no. Let me fix this. Why did this happen ? :(